#
# request 
#
bool command_request
---
#
# response: 1--> quiz; 2--> system; 3--> stop
#
int8 command_personality
int8 command_first
int8 command_start

